addons/strings/stringtable.xml

```diff
@@ -38,7 +38,7 @@
             <German>.</German>
             <Polish> </Polish>
             <Russian> </Russian>
-            <French> </French>
+            <French>,</French>
```

Comment:
Are you sure about this, it doesn't seem to match some sources
https://en.wikipedia.org/wiki/Decimal_separator

Reply:
I confused it with the decimal separator